core/src/main/java/org/eclipse/hono/tracing/TracingHelper.java

```diff
@@ -308,6 +312,71 @@ public static SpanContext extractSpanContext(final Tracer tracer, final MultiMap
         return tracer.extract(Format.Builtin.TEXT_MAP, new MultiMapExtractAdapter(headers));
     }
 
+    /**
+     * Injects a {@code SpanContext} as key-value pairs into a given operation.
+     * <p>
+     * This provides a generic way to serialize a span context in any kind of textual data.
+     * See {@link #extractSpanContext(Tracer, Supplier)} for the corresponding method to deserialize the
+     * context from that data.
+     *
+     * @param tracer The Tracer to use for injecting the context.
+     * @param spanContext The context to inject.
+     * @param keyValueConsumer The operation that will receive the key-value pairs representing the context.
+     * @throws NullPointerException if any of the parameters is {@code null}.
+     */
+    public static void injectSpanContext(final Tracer tracer, final SpanContext spanContext,
```

Comment:
where is this used?

Reply:
Currently nowhere. But that method makes sense as the counterpart to the `extractSpanContext(tracer, supplier)` method and may be useful in external code using `TracingHelper`.

Reply:
ok, just wanted to make sure I'm not missing something ;-)